tests/cases/fourslash/completionListAtIdentifierDefinitionLocations_interfaceMembers2.ts

```diff
@@ -1,7 +0,0 @@
-/// <reference path='fourslash.ts' />
-
-////var aa = 1;
-
-////interface a { f/*interfaceValue2*/
-
-goTo.eachMarker(() => verify.completionListIsEmpty());

```

Comment:
can you instead of deleting this have test case that verifies "aa" is not present in the completion

Reply:
There is a test in `completionsInterfaceElement` just like this.